tests/format/js/arrow-call/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -255,26 +249,23 @@ const testResults = results.testResults.map((testResult) =>
 );
 
 it("mocks regexp instances", () => {
-  expect(() =>
-    moduleMocker.generateFromMetadata(moduleMocker.getMetadata(/a/))
-  ).not.toThrow();
+  expect(() => moduleMocker.generateFromMetadata(moduleMocker.getMetadata(/a/)))
+    .not.toThrow();
 });
 
 expect(() => asyncRequest({ url: "/test-endpoint" })).toThrowError(
   /Required parameter/
 );
 
-expect(() =>
-  asyncRequest({ url: "/test-endpoint-but-with-a-long-url" })
-).toThrowError(/Required parameter/);
+expect(() => asyncRequest({ url: "/test-endpoint-but-with-a-long-url" }))
+  .toThrowError(/Required parameter/);
 
 expect(() =>
   asyncRequest({ url: "/test-endpoint-but-with-a-suuuuuuuuper-long-url" })
 ).toThrowError(/Required parameter/);
 
-expect(() =>
-  asyncRequest({ type: "foo", url: "/test-endpoint" })
-).not.toThrowError();
+expect(() => asyncRequest({ type: "foo", url: "/test-endpoint" })).not
+  .toThrowError();
```

Comment:
It looks to get worse to read.
In my opinion, this is an unfortunate edge case.
We need better heuristics, if this can't be accepted.

Reply:
We suppose to use installed v3.0.1 not the dev version to format, something is wrong.